<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettings</class>
 <widget class="QWidget" name="FormSettings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="GBox_ActiveMenuItem">
     <property name="title">
      <string>Active menu items:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableAnime">
        <property name="text">
         <string>Anime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableManga">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Manga</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableAMV">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>AMV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableDorama">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Dorama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableEditing">
        <property name="text">
         <string>Editing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBox_EnableLookLater">
        <property name="text">
         <string>Want to look</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBox_EditTags">
     <property name="title">
      <string>Tag list:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="LineEdit_AddTag">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="TButton_AddTag">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>To add in the list below, a tag from a field at the left</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="TButton_DeleteTag">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>To remove chosen in the list below a tag</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="ListView_Tags">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBox_FieldsForEdit">
     <property name="title">
      <string>Optional item:</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="CheckBox_Director">
        <property name="text">
         <string>Director</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBox_OrigTitle">
        <property name="text">
         <string>Original title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="CheckBox_PostScoring">
        <property name="text">
         <string>Postscoring</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="BtnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnBox</sender>
   <signal>rejected()</signal>
   <receiver>FormSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
